src/amp-story-player/amp-story-player-impl.js

```diff
@@ -967,11 +980,46 @@ export class AmpStoryPlayer {
 
     if (this.isDesktopPanelExperimentOn_) {
       this.checkButtonsDisabled_();
+      this.getUiState_();
```

Comment:
`getUiState_()` should be a getter with no side effects (updating the buttons). Please either rename the method so it better reflects what it does, or split the code to have a getter + update the state.